apps/wing/src/analytics/collecter.ts

```diff
@@ -0,0 +1,47 @@
+import { Command } from "commander";
+import { AnalyticsStorage } from "./storage";
+import { AnalyticEvent, GitData, NodeData, OSData } from "./event";
+import * as os from "os";
+import { PACKAGE_VERSION } from "../cli";
+import { getDevStats } from "./git-analysis";
+import { getCIData, isInCI } from "./ci-analysis";
+
+export class AnalyticsCollector {
```

Comment:
Not sold on the idea of leaving this as a class with static methods from a design perspective, but works for the testing phase.